template/codeland.rb

```diff
@@ -24,11 +24,16 @@ def rails_4?
   puts 'Please upgrade to ruby 2.2 or more'
 end
 
+gem_group :development do
+  gem 'rack-mini-profiler'
+  gem 'letter_opener'
+end
+
 gem_group :development, :test do
   unless rails_4?
     gem 'pry-rails', '~> 0.3.2'
   end
-  gem 'machinist', '~> 2.0'
+  gem 'factory_girl_rails', '~> 4.5.0'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.